src/decompiler-naming.h

```diff
@@ -172,23 +174,25 @@ void RenameAll(Module& module) {
   // We also filter common C++ keywords/STL idents that make for huge
   // identifiers.
   // FIXME: this can obviously give bad results if the input is not C++..
+  // clang-format off
   std::set<string_view> filter = {
-    { "const" },
-    { "std" },
-    { "allocator" },
-    { "char" },
-    { "basic" },
-    { "traits" },
-    { "wchar" },
-    { "t" },
-    { "void" },
-    { "int" },
-    { "unsigned" },
-    { "2" },
-    { "cxxabiv1" },
-    { "short" },
-    { "4096ul" },
+      {"const"},
```

Comment:
Strange that it still changed these lines despite being "off"?   Not a big deal but odd.

Reply:
Chromium style is 4-space indentation for array elements, so I preserved that part before re-breaking the lines and adding `// clang-format off`. We can make this array indentation to 2-spaces with another clang-format option though.